### PR TITLE
PT-1128: Update docs

### PR DESCRIPTION
**Pull request by krankenbro**

### CODE REVIEW

File: build/README.md [hunk 1]
```diff
@@ -129,3 +129,38 @@ Used to automate the routine operations with release branches
...
+Version number can be specified by PlatformVersion parameter
+For example:
+```console
+vc-build Init
+vc-build Init -PlatformVersion 3.52.0
+```
+#### Install
+Gets the -Module parameter, which is an array of module ids, and updates vc-package.json.
+Also module id can be  supplemented with the version number.
+Discovery and Probing directories can be overrided via -DiscoveryPath and -ProbingPath
```
Comment (eMazeika) [inline build/README.md:145]: overridden

File: build/README.md [hunk 2]
```diff
@@ -129,3 +129,38 @@ Used to automate the routine operations with release branches
...
+Examples:
+```console
+vc-build install -Module VirtoCommerce.Cart VirtoCommerce.Catalog
+vc-build install -Module VirtoCommerce.Cart VirtoCommerce.Catalog:3.38.0
```
Comment (eMazeika) [inline build/README.md:149]: should there be -DiscoveryPath and -ProbingPath parameters added?

File: build/README.md [hunk 3]
```diff
@@ -129,3 +129,38 @@ Used to automate the routine operations with release branches
...
+```
+#### InstallModules
+Installs modules according to vc-package.json and solves dependencies
```
Comment (eMazeika) [inline build/README.md:152]: solves - resolves

File: build/README.md [hunk 4]
```diff
@@ -129,3 +129,38 @@ Used to automate the routine operations with release branches
...
+```
+#### InstallPlatform
+Installs platform according to vc-package.json
+```console
+vc-build InstallPlatform
+```
+#### Uninstall
+Gets -Modules parameter and removes specified modules
+```console
+vc-build uninstall -Module VirtoCommerce.Cart VirtoCommerce.Catalog
```
Comment (eMazeika) [inline build/README.md:165]: -Modules or -Module?

File: build/README.md [hunk 5]
```diff
@@ -129,3 +129,38 @@ Used to automate the routine operations with release branches
 #### CompleteHotfix:
 - Merges hotfix branch into master
 - Adds tag and pushes
+
+## Packages management
+#### Init
+Creates vc-package.json boilerplate with the latest version number of the platform.
+Version number can be specified by PlatformVersion parameter
```
Comment (eMazeika) [inline build/README.md:136]: "PlatformVersion" -> "-PlatformVersion"

File: build/README.md (hunk 2 above)
Comment (krankenbro) [reply build/README.md:149]: These params are for InstallModules target. I added an example there.

Comment (krankenbro) [reply build/README.md:149]: I've added an extra example

**Pull request merged**